test/cppsuite/configs/burst_inserts_stress.txt

```diff
@@ -1,31 +1,43 @@
-# Used as a stress test for the framework.
+# Burst inserts is attempting to replicate conditions seen in WT-7798.
+# This workload is a highly stressful insert and read workload that performs insertions rapidly for
+# burst duration and then sleeps the op_rate defined in the insert config.
```

Comment:
`burst duration` -> Use `burst_duration` if you use `op_rate`.

Reply:
bde1106ec